src/rofl/common/crofsock.cc

```diff
@@ -1288,106 +1288,129 @@ void crofsock::handle_timeout(cthread &thread, uint32_t timer_id) {
   }
 }
 
-void crofsock::send_message(rofl::openflow::cofmsg *msg) {
+enum crofsock::msg_result_t crofsock::send_message(rofl::openflow::cofmsg *msg, bool enforce_queueing, bool keep_message) {
```

Comment:
`enum` keyword should be removed to be in line with other function returning `msg_result_t`